### PR TITLE
EPMEDU-602: Remove attached photo

### PR DESCRIPTION
**Pull request by madoyanzaven**
Remove attached photo

### CODE REVIEW

File: animeal.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2452,7 +2464,6 @@
 				6BB058E8BBC9A383D2FE14FF /* BankAccount.swift in Sources */,
 				F0C51AC9C71D7585539B42A1 /* Question+Schema.swift in Sources */,
 				D09306AD2965939F00BABB17 /* AttachPhotoViewItem.swift in Sources */,
-				DBB6994C2962CB240048E0CF /* FeedingSnapshotStore.swift in Sources */,
```
Comment (diantyn) [inline animeal.xcodeproj/project.pbxproj]: Is it expected?

File: animeal/src/Flows/AttachPhoto/Coordinator/AttachPhotoCoordinator.swift [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+                image: image
+            )
+            alertViewController.addAction(
+                AlertAction(title: L10n.Action.no, style: AlertAction.Style.inverted) {
+                    alertViewController.dismiss(animated: true)
+                }
+            )
+            alertViewController.addAction(
+                AlertAction(title: L10n.Action.yes, style: AlertAction.Style.accent) {
+                    self.deletePhotoEvent?()
```
Comment (diantyn) [inline animeal/src/Flows/AttachPhoto/Coordinator/AttachPhotoCoordinator.swift]: weak self

File: animeal/src/Flows/AttachPhoto/Coordinator/AttachPhotoCoordinator.swift [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    func routeTo(_ route: AttachPhotoRoute) {
+        switch route {
+        case .deletePhoto(let image):
+            let alertViewController = AlertViewController(
+                title: L10n.Attach.Photo.Delete.message,
+                image: image
+            )
+            alertViewController.addAction(
+                AlertAction(title: L10n.Action.no, style: AlertAction.Style.inverted) {
+                    alertViewController.dismiss(animated: true)
```
Comment (diantyn) [inline animeal/src/Flows/AttachPhoto/Coordinator/AttachPhotoCoordinator.swift]: weak alerViewController

File: animeal/res/ka.lproj/Localizable.strings [hunk 4]
```diff
@@ -43,7 +43,8 @@
 "action.copyIBAN"                                 = "კოპირება IBAN";
 "action.deleteAccount"                            = "Ანგარიშის წაშლა";
 "action.iWillFeed"                                = "ვაჭმევ";
-"action.finish"                                   = "";
+"action.yes"                                      = "Yes";
```
Comment (igorTimofiejczyk) [inline animeal/res/ka.lproj/Localizable.strings:46]: We need Georgian lang here, currently we are using google translate for that

Comment (madoyanzaven) [reply animeal/res/ka.lproj/Localizable.strings:46]: To be honest I put the same english text in Google sheet as well , for avoiding empty string at this moment))

**Review by igorTimofiejczyk — APPROVED** [review]

File: animeal/res/ka.lproj/Localizable.strings (hunk 4 above)
Comment (igorTimofiejczyk) [reply animeal/res/ka.lproj/Localizable.strings:46]: I will create a instruction on the kb page on how to work with string, so let's just skip it for now.

**Review by diantyn — APPROVED** [review]

**Pull request merged**